packages/web/src/components/Field.js

```diff
@@ -132,6 +135,45 @@ function Field(props) {
 		)
 	}
 
+	if (type === 'chromepicker'){
```

Comment:
We're not likely to use any other color pickers... would it make more sense to just set the `type` to `color`?

Reply:
I approve of using the html 5 color pickers over any kind of custom-built one - i found it's an oltion recently and it just works so much smoother than anything else i've seen before amd takes advantage of system-native color pickers. Especially neat on macOS, especially tedious in Firefox on windows.

Reply:
The `react-color` library actually works quite well across all platforms, and the native color picker can still crash Safari on macOS.